ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -1161,3 +1161,71 @@ presubmits:
     name: test_pull_request_origin_extended_networking
     rerun_command: /test extended_networking
     trigger: ((?m)^/test extended_networking,?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    optional: true
+    skip_report: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-origin-master-e2e-aws-cnv
+    rerun_command: /test e2e-aws-cnv
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws
+        - --template=/usr/local/e2e-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-origin-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-cnv
+        - name: TEST_COMMAND
+          value: /test.sh
+        - name: TEST_IMAGE
+          value: kubevirt/hco-tests:latest
```

Comment:
was something added elsewhere that is building this image?

Reply:
oh i see, this is being hosted on dockerhub.  i think we should do this the "right" way and introduce logic that will build the hco-tests image from a repo on each job run, the way our mainline e2e tests work.

Otherwise you're going to have a hard time synchronizing delivering test+code changes, or knowing why a job failed when you can't confirm what version of the tests it ran against.

Reply:
I don't want to stick with `:latest` tag, I need to improve that on the hco side. The issue in my opinion is that there are no releases right now in the hco repo. I am not sure if we can properly solve the scenario with one job. I think we would need two: One which tests latest master (maybe like you suggest) and one which tests the latest release. I would not focus on this image too much for the initial version, if that is fine for you.